pkg/rbac/tokenReview_test.go

```diff
@@ -51,40 +64,45 @@ func Test_IsValidToken_usingCache(t *testing.T) {
 
 // TokenReview in cache is older than 60 seconds.
 func Test_IsValidToken_expiredCache(t *testing.T) {
-	cache := Cache{
-		tokenReviews:        map[string]*tokenReviewResult{},
-		tokenReviewsPending: map[string][]chan *tokenReviewResult{},
-		tokenReviewsLock:    sync.Mutex{},
-	}
+	// Initialize cache state - cache updated 2 minutes ago.
+	cache := newCache()
 	cache.tokenReviews["1234567890"] = &tokenReviewResult{
-		updatedAt:   time.Now().Add(time.Duration(-5) * time.Minute),
-		tokenReview: &authv1.TokenReview{},
+		updatedAt: time.Now().Add(time.Duration(-2) * time.Minute),
+		tokenReview: &authv1.TokenReview{
+			Status: authv1.TokenReviewStatus{
+				Authenticated: true,
+			},
+		},
 	}
 
+	// Execute function
 	result, err := cache.IsValidToken(context.TODO(), "1234567890")
 
+	// Validate results
 	if result {
-		t.Error("Expected token to resolve to valid.")
+		t.Error("Expected token to be invalid.")
 	}
 	if err != nil {
 		t.Error("Received unexpected error from IsValidToken()", err)
 	}
-	// TODO: Verify that cache got updated.
-	if cache.tokenRevews["1234567890"].updatedAt
+	// Verify that cache was updated within the last 1 millisecond.
+	if cache.tokenReviews["1234567890"].updatedAt.Before(time.Now().Add(time.Duration(-1) * time.Millisecond)) {
+		t.Error("Expected the cached TokenReview to be updated within the last millisecond.")
```

Comment:
"within the last minute"

Reply:
Actually `millisecond` is correct here.  This verifies that the function found that the cache had expired and it updated it with new results.